### DISCUSSION TITLE
Set cluster name different from the rabbitmq crd name

### DISCUSSION THREAD

**Discussion by kotyara85** [rabbitmq/cluster-operator#1590] [General]
Hey there,
I do have a very specific case where the rmq cluster is named the same within each namespace(to simplify deployments), but this leads to all of them having the same name in the config.

Is it possible to have the same cluster name in the operator config, but set diffirent cluster names for the actual cluster configuration?
Thanks!

**Comment by mkuratczyk** [comment]
You can set `cluster_name` in `.spec.rabbitmq.additionalConfig` to whatever you want.

**Reply by mkuratczyk** [reply]
And specifically, you can use environment variable interpolation to have the same value set for all your clusters in the deployment YAML but have a different name at runtime. Something like:
```
spec:
  rabbitmq:
    additionalConfig: |
      cluster_name: foo-$(MY_POD_NAMESPACE)
```
so that the namespace will be appended to each cluster's name.
